spec/controllers/api_blocks_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe Api::BlocksController do
```

Comment:
Maybe not important, but controller tests are no longer recommended (and I'm surprised this worked without the [rails-controller-testing](https://github.com/rails/rails-controller-testing) gem). It looks like you created a request spec in spirit anyway.

Reply:
They are now request specs!